example/pipeline/pipeline_txt.ipynb

```diff
@@ -46,7 +46,7 @@
     }
    ],
    "source": [
-    "from uniflow.pipeline import Pipeline\n",
+    "from uniflow.pipeline import MultiProcessPipeline\n",
```

Comment:
nit: this name can be misleading because people will think that you are using multiple processes while we are using multiple thread pool.